sdk/tables/perf-tests/data-tables/tsconfig.json

```
@@ -0,0 +1,11 @@
+{
+  "extends": "../../../../tsconfig.package",
+  "compilerOptions": {
+    "module": "CommonJS",
+    "outDir": "./dist-esm",
```

You can use `ts-node` so no need to compile:
```suggestion
    "noEmit": true,
```